src/TLE.ts

```diff
@@ -1252,15 +1247,13 @@ export class Token {
     private _span: language.Span;
     private _stringValue: string;
 
-    private static create(tokenType: TokenType, startIndex: number, stringValue: string): Token {
+    public constructor(tokenType: TokenType, startIndex: number, stringValue: string) {
```

Comment:
>tokenType: TokenType, startIndex: number, stringValue: string [](start = 23, length = 61)

Nit: Why not use property definition via constructor args here live you've done in the Tokenizer constructor above? I don't have strong feelings about it, but it seems it would be good to be consistent in style.

Reply:
Not too worried on consistency for that.  I tend to use it for new code, but none of the old code has it.  Changing over everything would be a lot of work.